src/v2/Apps/Collect/Routes/Collect/Utils/getMetadata.ts

```diff
@@ -15,21 +15,19 @@ export type Medium =
 
 export type Color =
   | "black-and-white"
-  | "darkblue"
-  | "darkgreen"
-  | "darkviolet"
+  | "blue"
   | "gold"
-  | "lightblue"
-  | "lightgreen"
+  | "gray"
+  | "green"
   | "orange"
   | "pink"
+  | "purple"
   | "red"
-  | "violet"
   | "yellow"
 
 interface Props {
-  medium: Medium
-  color: Color
+  medium: Medium | null
```

Comment:
@mdole could you elaborate on this change? Why are we doing them optional?

Reply:
good question @nickskalkin! I originally added the `| null` because that's how they were being [passed in the tests](https://github.com/artsy/force/blob/b149001f55aefda4e311d252a5b67dcb8e8be1c3/src/v2/Apps/Collect/Routes/Collect/Utils/__tests__/getMetadata.jest.ts#L8).

Your comment encouraged me to make sure I understood what was happening here. It turns out that the color/medium values defined in `getMetadata` are used for these "alternate routes" for the /collect page. For example, you can hit https://www.artsy.net/collect/painting or https://www.artsy.net/collect/color/yellow.

I'm guessing those exist for SEO reasons. Either way, it turns out that if the user _doesn't_ hit one of those routes but just hits the collect page normally (in other words, https://www.artsy.net/collect/ or https://www.artsy.net/collect/painting?colors%5B0%5D=yellow), then `getMetadata()` will return `undefined` for both color and medium.

In summary: these types should be `| undefined` because they will often _be_ undefined. I just pushed up another commit that changes this to `| undefined` and updates the test as well.

Thanks for flagging! Would have been misleading to introduce `| null` here :)